src/main/webapp/style/abstracts/_utilities.css

```diff
@@ -29,7 +29,8 @@
 }
 
 /*
-  Utility to vertically contents of a container
+  Utility to vertically contents of a container.
```

Comment:
extreme nit: did you mean `... vertically align contents ...`

Reply:
I did, but I deleted the utility, so this is also deleted now.